File: alafia-server/src/main/java/es/alafia/server/model/Course.java
package es.alafia.server.model;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Course {

    @Id
    private String id;

    private String description;

    private Double price;

    private String nutritionalValues;

    private String obtainProcess;

    private String history;

    private String urlVideo;
}

File: alafia-web-application/src/app/model/course.ts
export class Course {
  id: string;
  description: string;
  price: number;
  nutritionalValues: string;
  obtainProcess: string;
  history: string;
  served: boolean;
  urlVideo: string;

  constructor(id: string,
              description: string,
              price: number,
              nutritionalValues: string,
              obtainProcess: string,
              history: string,
              urlVideo: string) {
    this.id = id;
    this.description = description;
    this.price = price;
    this.nutritionalValues = nutritionalValues;
    this.obtainProcess = obtainProcess;
    this.history = history;
    this.served = false;
    this.urlVideo = urlVideo;
  }
}
